### PR TITLE
Add fab commands to backup and rebuild db

### PR DESCRIPTION
**Pull request by kashizui**
Used these commands to rebuild the db on the production and staging
servers after enabling innodb_file_per_table.

@percyliang

### CODE REVIEW

File: deployment/fabfile.py [hunk 1]
```diff
@@ -414,6 +414,90 @@ def maintenance(mode):
...
+    run('mysqldump --user={user} --password={password} '
+        '--single-transaction --quick {database} > {out}'.format(
+            database=db_name, user=db_user, password=db_password, out=dumpfile),
+        shell=True)
+    return dumpfile
+
+
+@roles('web')
+@task
+def rebuild_db(backupdir='backup'):
```
Comment (percyliang) [inline deployment/fabfile.py]: Maybe we should add instructions on how to make sure that innodb_file_per_table is always on (in which case you don't need to ever run these commands?)  Provide instructions on how to run these commands?

**Comment by percyliang** [comment]
Even just saying that it's a boolean config option in the MySQL config file
is useful + the fact the default is false, and you should set it to true to
avoid fragmentation?  In general, to the extent that we can capture
institutional knowledge in written form, we should try to do it.

**Comment by kashizui** [comment]
That's a good point -- will do.

**Comment by kashizui** [comment]
@percyliang Added some documentation

**Comment by percyliang** [comment]
Thanks!
LGTM

**Pull request merged**